lib/services/groups/groupRegistryMemory.js

```diff
@@ -26,6 +26,7 @@ var registeredGroups = {},
     logger = require('logops'),
```

Comment:
Please add an entry in CHANGES_NEXT_RELEASE corresponding to the fix done in this PR.